js/map_utils.js

```diff
@@ -134,12 +134,14 @@ function updateView(newView, fireAnalytics) {
     backgroundState(otherstates, scale = zoom_scale);
     foregroundState(thisstate, scale = zoom_scale);
     
-    statecounties
-      .transition()
-      .duration(500) 
-      .style("stroke-width",  0.75/zoom_scale); // make all counties have scaled stroke-width
+    
   }
-
+  var allcounties = d3.selectAll('.county');
```

Comment:
@aappling-usgs has noted this to me before, but I think we would be unnecessarily selecting and changing all 3224 counties when we only need to change the stroke of the current state's counties. This might affect performance.

Reply:
I did this on purpose - the alternative is that the other state county doesn't start scaled the right way when panned to. To see this check dev and zoom to NJ, then click NY and see the borders start huge.

Reply:
Hmmm OK. I see now. I wonder if we can just chop off the transition duration instead?

Reply:
I tried it several ways, including chopping the duration to 0 (side note, if you do that, don't you want to use D3 to set the style `.style("stroke-width"..)` and not use a transition at all?) and comparing performance of the whole US vs just the counties of the highlighted states. 

I couldn't see an impact of either of the approaches on performance, but cutting off the transition does leave the somewhat undesired artifact of having the county you clicked on to zoom out immediately get thick while other things smooth through. 

I looked on chrome (and throttled the CPU) and tried this on FF visually. But all of these are on mac and I sense windows may be different and if we are worried about that we should try it out. It ends up being a matter of using `allcounties` or `statecounties` and whether the transition only happens within the `if(activeView !== 'USA')` or regardless in case someone wants to give it a spin.

Reply:
if you didn't notice any significant performance issues, then I'm fine with how you've implemented it! It does look smoother overall.